libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1708,21 +1568,21 @@ void WaveTrack::InsertSilence(double t, double len)
          // use No-fail-guarantee
          pChannel->InsertClip(move(clip));
       }
-      else {
+      else
+      {
          // Assume at most one clip contains t
          const auto end = clips.end();
          const auto it = std::find_if(clips.begin(), end,
-            [&](const WaveClipHolder &clip) { return clip->WithinPlayRegion(t); } );
+            [&](const WaveClipHolder &clip) { return clip->SplitsPlayRegion(t); } );
 
          // use Strong-guarantee
          if (it != end)
             it->get()->InsertSilence(t, len);
 
          // use No-fail-guarantee
-         for (const auto &clip : clips) {
-            if (clip->BeforePlayStartTime(t))
+         for (const auto &clip : mClips)
```

Comment:
This is an error in conflict resolution!  Shift the clips of each channel, not only those of `this` `WaveTrack`.

Reply:
Oj ... 🥇